source/features/search-releases.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+
+import features from '../libs/features';
+import {getOwnerAndRepo} from '../libs/utils';
+import {octoface} from '../libs/icons';
+
+async function init() {
+	if (!hasAnyReleases()) {
+		return false;
+	}
+
+	addSelectMenu();
+	registerEventListener();
+}
+
+const hasAnyReleases = () => select.exists('.release-entry') || select.exists('.release');
+
+const addSelectMenu = () => {
+	const container = select('.subnav');
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const selectedTag = getSelectedTagFromUrl();
+
+	container.append(
+		<div class="float-right d-flex flex-shrink-0 flex-items-center mb-3">
+			<details class="details-reset details-overlay select-menu branch-select-menu">
+				<summary class="btn btn-sm select-menu-button css-truncate" data-hotkey="w" title="Filter releases" aria-haspopup="menu">
+					<i>Filter:</i>&nbsp;
+					<span class="css-truncate-target">{selectedTag || 'Releases'}</span>&nbsp;
+				</summary>
+				<details-menu
+					class="rgh-release-details select-menu-modal position-absolute"
+					src={`/${ownerName}/${repoName}/ref-list/master?source_action=disambiguate&source_controller=files`}
+					preload
+					role="menu"
+					style={{zIndex: 99}}
+				>
+					<include-fragment class="select-menu-loading-overlay anim-pulse">
+						{octoface()}
+					</include-fragment>
+				</details-menu>
+			</details>
+		</div>
+	);
+};
+
+const getSelectedTagFromUrl = () => {
+	const [tag = null] = location.href.match(/(?<=\/releases\/tag\/)([^/]+)/) || [];
```

Comment:
```suggestion
	const [tag] = location.href.match(/(?<=\/releases\/tag\/)([^/]+)/) || [undefined];
```

I don't know if this works, but it would be preferred if it does: https://github.com/sindresorhus/meta/issues/7

Reply:
It works. Suggestion implemented.

Reply:
Yeah I hate this specific nitpick of TypeScript, I wish they had a better suggestion in the error message.

Reply:
I wish! But anyways I love Typescript! 🤪